spark/sql/functions/generated.go

```diff
@@ -3073,9 +3073,14 @@ func BitmapOrAgg(col column.Column) column.Column {
 	return column.NewColumn(column.NewUnresolvedFunctionWithColumns("bitmap_or_agg", col))
 }
 
-// Ignore UDF: call_udf: (udfName: str, *cols: 'ColumnOrName') -> pyspark.sql.connect.column.Column
+// Ignore UDF: call_udf: (udfName: str, *cols: 'ColumnOrName') -> sql.connect.column.Column
 
-// Ignore UDT: unwrap_udt: (col: 'ColumnOrName') -> pyspark.sql.connect.column.Column
+// UnwrapUdt: (col: 'ColumnOrName') -> sql.connect.column.Column
+//
+// Unwraps the UDT so you can work with the internal data structure embedded in the type
+func UnwrapUdt(col column.Column) column.Column {
+	return column.NewColumn(column.NewUnresolvedFunctionWithColumns("unwrap_udt", col))
+}
```

Comment:
This file is generated using a script, can you please move this implementation into a dedicated file so that it doesn't get overridden once we regenerate the code?

Reply:
Of course, sorry for that. I never even considered to take a look at the file which tells quite clearly that the file is generated.

Reply:
Ideally this stub should be generated rather than coded. What drives the generation?

Reply:
I recommend we reject this PR and rerun and the generation script instead. I have looked at gen.py and I don't see a reason why this would not be generated. It is a valid and important function even though we might not have UDT support yet since the data source may very well use UDT's and the function returns the internal data structure of the type rather than the type making it usable from regular spark.

Reply:
@magpierre I am very interested to help out on this and other future works 

Please feel free to drop me `elbeherymustafa@gmail.com` 

thanks